src/table/__tests__/resizable-columns.test.tsx

```diff
@@ -500,29 +496,157 @@ describe('resize in rtl', () => {
-    fireMouseMove(-10);
-    fireMouseup(-10);
+    firePointerdown(wrapper.findColumnResizer(1)!);
+    firePointermove(-10);
+    firePointerup(-10);
     expect(wrapper.findColumnHeaders()[0].getElement()).toHaveStyle({ width: '80px' });
   });
 });
+
+describe('UAP buttons', () => {
+  // Makes the drag buttons (which are positioned in a portal) easier to find if there's only one set.
+  const singleColumnDefinition = [{ id: 'id', header: 'Id', cell: (item: any) => item.id, width: 150, minWidth: 80 }];
+
+  let mockWidth = 150;
+
+  const originalBoundingClientRect = HTMLElement.prototype.getBoundingClientRect;
+  beforeEach(() => {
+    HTMLElement.prototype.getBoundingClientRect = function () {
+      const rect = originalBoundingClientRect.apply(this);
+      if (this.tagName === 'TH') {
+        rect.width = mockWidth;
+      }
+      return rect;
+    };
+  });
+
+  afterEach(() => {
+    mockWidth = 150;
+    HTMLElement.prototype.getBoundingClientRect = originalBoundingClientRect;
+  });
+
+  test('hides UAP buttons by default', () => {
+    renderTable(<Table {...defaultProps} columnDefinitions={singleColumnDefinition} />);
+    expect(findDragHandle().findVisibleDirectionButtonInlineStart()).toBeNull();
+    expect(findDragHandle().findVisibleDirectionButtonInlineEnd()).toBeNull();
+  });
+
+  test('shows UAP buttons when clicked', () => {
+    const { wrapper } = renderTable(<Table {...defaultProps} columnDefinitions={singleColumnDefinition} />);
+    firePointerdown(wrapper.findColumnResizer(1)!);
+    firePointerup(0);
+
+    expect(findDragHandle().findVisibleDirectionButtonInlineStart()).toBeTruthy();
+    expect(findDragHandle().findVisibleDirectionButtonInlineEnd()).toBeTruthy();
+  });
+
+  test('shows UAP buttons when activated with keyboard', () => {
+    const { wrapper } = renderTable(<Table {...defaultProps} columnDefinitions={singleColumnDefinition} />);
+
+    wrapper.findColumnResizer(1)!.focus();
+    wrapper.findColumnResizer(1)!.keydown(KeyCode.enter);
+
+    expect(findDragHandle().findVisibleDirectionButtonInlineStart()).toBeTruthy();
+    expect(findDragHandle().findVisibleDirectionButtonInlineEnd()).toBeTruthy();
+  });
+
+  test('hides UAP buttons when Escape is pressed after a pointer interaction', () => {
+    const { wrapper } = renderTable(<Table {...defaultProps} columnDefinitions={singleColumnDefinition} />);
+
+    wrapper.findColumnResizer(1)!.click();
+    wrapper.findColumnResizer(1)!.focus();
+    expect(findDragHandle().findVisibleDirectionButtonInlineStart()).toBeTruthy();
+    expect(findDragHandle().findVisibleDirectionButtonInlineEnd()).toBeTruthy();
+
+    wrapper.findColumnResizer(1)!.keydown(KeyCode.escape);
+    expect(findDragHandle().findVisibleDirectionButtonInlineStart()).toBeNull();
+    expect(findDragHandle().findVisibleDirectionButtonInlineEnd()).toBeNull();
+  });
+
+  test('hides UAP buttons when arrow keys are pressed before the resizer button is activated', () => {
```

Comment:
This is different to the behavior in other components, e.g. app layout panels, sortable list. Is there a strong reason why it needs to be different here?